### PR TITLE
[LOCALSPL] Don't hardcode C:\ReactOS

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-14747](https://jira.reactos.org/browse/CORE-14747)

### CODE REVIEW

File: win32ss/printing/providers/localspl/printerdrivers.c [hunk 1]
```diff
@@ -217,12 +214,11 @@ _LocalGetPrinterDriverLevel5(PLOCAL_PRINTER_HANDLE pHandle, PDRIVER_INFO_5W* ppD
     DWORD n;
     PCWSTR pwszStrings[5];
 
-    /* Clearly these things should not be hardcoded, so when it is needed, someone can add meaningfull values here */
     pwszStrings[0] = pHandle->pPrinter->pwszPrinterDriver;  // pName
-    pwszStrings[1] = wszCurrentEnvironment;  // pEnvironment
```
Comment (SergeGautherie) [inline win32ss/printing/providers/localspl/printerdrivers.c]: Unwanted change?

Comment (katahiromz) [reply win32ss/printing/providers/localspl/printerdrivers.c]: Reverted.

File: win32ss/printing/providers/localspl/main.c [hunk 2]
```diff
@@ -134,6 +136,19 @@ _InitializeLocalSpooler(void)
...
+    WCHAR szSysDir[MAX_PATH];
+
+    if (!GetSystemDirectoryW(szSysDir, _countof(szSysDir)))
+    {
+        ERR("GetSystemDirectoryW failed\n");
+        return FALSE;
+    }
+
+    lstrcpyW(wszLocalSplFile, szSysDir);
+    lstrcatW(wszLocalSplFile, L"\\localspl.dll");
```
Comment (ThFabba) [inline win32ss/printing/providers/localspl/main.c]: strsafe please

Comment (katahiromz) [reply win32ss/printing/providers/localspl/main.c]: Used `<strsafe.h>`.

**Pull request merged**